src/instructlab/configuration.py

```diff
@@ -148,6 +149,38 @@ def api_base(self):
         """Returns server API URL, based on the configured host and port"""
         return get_api_base(self.host_port)
 
+class _deepspeed(BaseModel):
```

Comment:
For local dev, I recommend cloning the current interface PR https://github.com/instructlab/training/pull/12, installing locally via `pip install -e .` and then importing these directly.

Reply:
yep, I am removing these and opting to use the ones you introduced. Still valuable to have a top level `_train` class in here though